README.md

```diff
@@ -14,7 +14,7 @@ We use [terraform](https://www.terraform.io/) for provisioning servers and [ansi
 To set up a new deployment of Simple on AWS, follow these steps:
 
 1. Use the [`terraform/`](/terraform) documentation to provision the necessary infrastructure on AWS
-1. Use the [`ansible/`](/ansible) documentation to set up simple-server on the AWS infrastructure
+1. Use the [`ansible/`](/ansible) documentation to configure your AWS infrastructure
```

Comment:
The previous wording seems clearer, configure AWS infrastructure can also imply configuring instance sizes etc.

Reply:
Ah good point. I guess the confusion I was trying to get _away_ from is - This doesn't ship Simple code onto the boxes. It just installs dependencies. How can we word that elegantly?

Reply:
Something like this?
- To prepare your infrastructure for setting up `simple-server`
- To install dependencies required to run `simple-server`

Reply:
I like `install dependencies`.